cortex/secondary/inactive_duration.py

```diff
@@ -187,6 +187,7 @@ def get_acc_bouts(acc_df, jerk_threshold=0.5):
     """
     acc_df['above_threshold'] = acc_df['acc_jerk'] > jerk_threshold
     acc_df['prev_above'] = acc_df['above_threshold'].shift()
+    acc_df = acc_df.dropna()
```

Comment:
As we know that the only None should be the first element from the shift() operation it may be more efficient to just remove that first data point. I'm not familiar with pandas optimizations for dropna(). 

(This issue has likely arisen since we updated pandas on the instances in May after this code was written).